kubernetes/samples/scripts/terraform/oke.create.sh

```diff
@@ -37,12 +37,13 @@ function setupTerraform () {
       curl -O https://releases.hashicorp.com/terraform/0.11.10/terraform_0.11.10_darwin_amd64.zip
       unzip terraform_0.11.10_darwin_amd64.zip
     elif [[ "${OSTYPE}" == "linux"* ]]; then
-       curl -O https://releases.hashicorp.com/terraform/0.11.8/terraform_0.11.8_linux_amd64.zip
-       unzip terraform_0.11.8_linux_amd64.zip
+#setup terraform#curl -LO --retry 3 https://releases.hashicorp.com/terraform/0.11.8/terraform_0.11.8_linux_amd64.zip#unzip -o terraform_0.11.8_linux_amd64.zip -d ${WORKSPACE}/bin/#rm terraform_0.11.8_linux_amd64.zip#chmod +x ${WORKSPACE}/bin/terraform
```

Comment:
Can you put this to multiple lines or just leave "#setup terraform"?

Reply:
removed commented out lines